dockerfiles/che-launcher/launcher_funcs.sh

```diff
@@ -117,6 +117,14 @@ get_docker_host_ip() {
             awk '{ print $2}'
 }
 
+get_docker_host_os() {
+  echo $(docker info | grep "Operating System:" | sed "s/^Operating System: //")  
+}
+
+get_docker_daemon_version() {
+  echo $(docker version | grep "Server version:" | sed "s/^Server version: //")
```

Comment:
This command returned an empty string for me. Using docker info and capital V in version worked:
`docker info | grep "Server Version:" | sed "s/^Server version: //"`